### DISCUSSION TITLE
Video Kit + Crypto + DDR Enabled Documentation or Demo, Yet?

### DISCUSSION THREAD

**Discussion by haroldnelson-2021** [polarfire-soc/polarfire-soc-documentation#375] [Applications and demos]
Do we have a Crypto example available on the PF SOC Video kit that has the DDR enabled?

Thanks.

**Comment by rockoguido** [comment]
Hi Harold,  I have some crypto examples, not sure if they execute outside of the LIM?  Involves the crypto lib, of course, which one has to go through a few hoops to get.   I suspect you already have this, you can ping me and we can compare notes.  
Dave

**Comment by nitindeshpande** [comment]
Hi,
 The user crypto bare metal projects are available [here](https://github.com/polarfire-soc/polarfire-soc-bare-metal-examples/tree/main/applications/user-crypto).
 
There are multiple build configurations provided in the SoftConsole project such as LIM-Release, DDR-Release. In some of them the DDR is enabled.

This [document](https://github.com/polarfire-soc/polarfire-soc-bare-metal-examples/tree/main) explains the build configurations.

Hope this helps.

Regards,
